tests/Cms/Collections/CollectionTestCase.php

```diff
@@ -7,11 +7,12 @@
 class CollectionTestCase extends TestCase
 {
 	public $collection = null;
+	public $collectionOptions = [];
```

Comment:
Could you please add unit tests for the new behavior?

- Passing an option to a collection and using it inside the collection
- Trying to access an option inside a collection that was not passed (should be provided to the collection function as `null`)
- Trying to override a core variable like `$site` with an option (which should be ignored)